src/lexer.c

```diff
@@ -119,7 +119,7 @@ int is_alnum(char c)
 
 int is_digit(char c)
 {
-    return (c >= '0' && c <= '9') ? 1 : 0;
+    return (c >= '0' && c <= '9');
```

Comment:
Remove parentheses as well since they does not effect expression precedence.

Reply:
Okay, I will modify that function and the function like that, is_whitespace and is_newline

Reply:
I push a new commit c818075e8ff887b81777387805245d294199626b to remove the redundant parentheses.